### ISSUE TITLE
Partenariat prospectif avec CAD?

### ISSUE THREAD

**Issue by DewiBrunet** [openfab-lab/openfab#210]
- [ ] ref to file .md or pullRequest #: .................................................  
*(à remplir quand l'issue est résolue)*
## contexte
Samedi dernier avec Chloé on a été faire un tour au C.A.D. (école supérieure privée d’art) pour le 
 www.caddesignfair.com/  (OpenFab a été invité car il y a quelques étudiants qui sont passés pour fabriquer des choses).

Les étudiants présents sur place m'on indiqué qu'il y avait pas vraiment d'atelier de fabrication au CAD, que c'était à eux d'en trouver un.

Une des personnes en charge de la logistique/communication a confirmé ce point en rajoutant un regret que parfois les étudiants font appel à une boite de service et qu'ils n’apprennent pas à faire.

## objectif
Et si OpenFab devenait un partenaire privilégié du CAD?

Pour le CAD: les étudiants auraient un accès à un lieu de fabrication, des conseils voir des stages/initiations

Pour OpenFab: ça pourrait nous faire des revenus supplémentaires réguliers, une ouverture vers le monde du design et de la vente d'objet, des nouveaux membres

## proposition
S'inspirer de ce qu'est en train de mettre en place le MAD?
Un système de plage horaire bloqué?
Des crédits offert sur 6 mois?
Des séances d'initiation?
Du coaching en prototypage?

--> Développer un "pack fablab" à proposer à d'autres organismes de design?
![untitled-1 copy](https://user-images.githubusercontent.com/25649502/39705915-0bc682be-5210-11e8-995b-141ca01e0fbd.jpg)

**Comment by DewiBrunet** [comment]
Yes, du coup développer un pack de base avec cours théorique/initiation machine puis des add-on sous forme de tokens ou workshops spécifiques ou accompagnement au prototypage...

- Ecole supérieur ça peux rentrer dans fabmob? même si c'est privé? (du coup leur proposer sur l'espace des deux ans des formations subsidiées?)
- On a un contact spécifique au CAD?

**Comment by nicolasdb** [comment]
Écoles sup. Non, max 18ans.
Pas de contact spécial encore. 

De nom, on  a un Quentin dans les membres. Oublié son nom de famille. Je pense qu'il a posé des questions sur la page openfab. @zuperninja anca, tu peux vérifier ? 

Vous n'avez pas causé à des profs?

**Comment by zuperninja** [comment]
Euh non je vois pas qui c'est..... Enfin pas de quentin du CAD

**Comment by nicolasdb** [comment]
Ok j'ai retrouvé.
Quentin jager

**Comment by nicolasdb** [comment]
@satblip m'a parlé de StMichel, ils ont investit dans du matos mais maintenant ça fait chier les profs qui n'ont pas de contenu ni le tps de s'en occuper. 

_(C'est bien ça Louis?)_

**Comment by DewiBrunet** [comment]
Bien bien bien

![3059_135068129530](https://user-images.githubusercontent.com/25649502/41429695-1c3be24e-700e-11e8-9cbd-7177069c152b.jpg)

On en a rapidement discuté, j'aimerai qu'on puisse se poser et établir une stratégie la dessus après les grandes vacances (parce que la les écoles vont être out jusque fin août). En mode quelle sont nos propositions (adaptables), les publics cibles, les tarifs, la liste des contacts... 

puis bam, on démarche, on a plein de réponse et on choisi les meilleurs!

![425241](https://user-images.githubusercontent.com/25649502/41430159-8ec5c39c-700f-11e8-9b4f-6c13b416d4f2.jpg)
"On t'as mis un 5 car tes tarifs sont trop bas, ta démarche pédagogique est contradictoire avec les valeurs OpenFab et puis le café était trop sucré".

**Comment by satblip** [comment]
@nicolasdb c'est exactement ça, du budget, du matos mais pas de soutient pédagogique

**Comment by nicolasdb** [comment]
Qu'est-ce qu'on fait de cette stratégie? 
On y reviendra quand on aura le tps? prochaine opportunité? 
On est peut-être pas prêt si ça n'avance pas.

Je clos l'issue?

**Comment by DewiBrunet** [comment]
Non, je suis la tête dans le day to day comme toi Nico et j'ai du mal à faire de la stratégie... mais ça fait partie des trucs important à faire!
On envisage une "team stratégie"?

**Comment by nicolasdb** [comment]
Je sais ;) mais y a tellement de vieux sujet en issue encore open.
Je me remotive à suivre le "getting things done" https://www.youtube.com/watch?v=7ZXyJAVqhzo

Il faut qu'on organise les sujets et les ranger dans les bonnes boites pour les traiter au bon moment.
Je pense que la stratégie sera dans les rôles du CA . Il faut alors une boite sujet stratégique qu'on ouvre quand on cause en CA. 
On peut alors la planifier, l'assigner à quelqu'un et l'ajouter dans les task du day2day opérationnel. 

d'où ma question sur l'issue "nouveau CA"
https://github.com/openfab-lab/openfab/issues/244#issuecomment-474275064

**Comment by nicolasdb** [comment]
Reçu sur contact. 
https://cad.be/fr/nouvelle-section-au-cad-fashion-textile-design/
